docs/src/surfaces/card/card.jsx

```diff
@@ -275,21 +278,22 @@ export default class ViewsTitleBar extends React.Component {
             return (
                 <div>
                     <Card>
-                        <Header size="large">Props</Header>
+                        <Typography size="large">Props</Typography>
 
                         <TableProps props={props} />
                     </Card>
 
                     {/* Card */}
-                    <Header size="large" style={{ marginTop: '55px' }} sub>
+                    <Heading variant="h2">
                         Card
-                        <Header.Subheader>
-                            A basic card has a faint border wrapping it's contents. It's box model is fluid so that it can adapt to it's parent container's width.
-                        </Header.Subheader>
-                    </Header>
+                    </Heading>
+
+                    <Typography variant="body1">
+                        A basic card has a faint border wrapping it's contents. It's box model is fluid so that it can adapt to it's parent container's width.
```

Comment:
Hey, here's the original! 😁 
Let's fix up the "it's" => "its" issue!

Reply:
Fixed.

Reply:
Looks to have been fixed as requested (definitely here, and in other places I notice as I'm scanning the updated version of this file).  Resolving.